### DISCUSSION TITLE
The rule use-button-type should include <Button /> components too

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [biomejs/biome#1568] [Ideas]
The rule only include <button /> tags

https://biomejs.dev/linter/rules/use-button-type/

**Comment by ematipico** [comment]
Can you provide more arguments for why a user component should be tracked by the rule? 

Users can name their components as they want, with the props that they want, making this impossible to track

**Reply by juliomuhlbauer** [reply]
I think this would mostly apply to users using a library like Park UI or Chakra UI, because the button is normally named Button in this libs. For sure there will be edge cases, but in most cases this rule would help a lot applying both button and Button

For example the Button component of Park UI: https://park-ui.com/docs/panda/components/button

Multiple times I accidentally broke my forms because I had other Buttons components on the form and the lint rule didn't detected.
